virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -1634,10 +1638,10 @@ You configure the persistent volume claim (PVC) of the bootable volume by editin
 |Description
 
 |*Name*
-|PVC name.
+|DataSource name.
```

Comment:
For every general reference to an API object (data source, instance type, virtual machine), the formatting should follow these API guidelines: 

"[API object formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references)

For terms that are API objects, the way they are written depends on whether the term is a general reference or an actual reference to the object.

General references

A general reference is any time you are speaking conceptually, or generally, about these components in a cluster.

When referring to API object terms in general usage, use lowercase and separate multi-word API objects. Default to following this guidance unless you are specifically interacting with/referring to the API object (see [Object references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references))."

This comment applies to each API object instance in the table descriptions.

Reply:
Hey @danielclowers this comment is still unresolved; in the descriptions where you're using general references to API objects, they should be lowercase and have spaces (data source, instance type, etc.)

Reply:
overlooked this. fixing!